ui/src/components/Publication/SidebarCard/Actions/index.tsx

```diff
@@ -283,15 +76,6 @@ const Actions: React.FC<ActionProps> = (props): React.ReactElement => {
     };
 
     // Adds a small delay so the error just instantly jump off the screen
-    React.useEffect(() => {
-        if (!showRatingsModel) setTimeout(() => setError(undefined), 500);
-        if (!showRedFlagModel) {
```

Comment:
Looks like 288 to 293 relates to Red Flags? Think it might only be added animation so removing it doesn't affect any functionality. Comment on line 285 could be removed if all the animation in useEffect is removed, or kept in if the RedFlag animation is kept in?